app/assets/javascripts/angularjs.coffee

```diff
@@ -0,0 +1,118 @@
+app = angular.module('treeApp', ['ngResource'])
+
+# order resource
+
+app.factory('Order', ['$resource', ($resource) ->
```

Comment:
you should start splitting this stuff out into multiple files, they're getting pretty big.